pkg/machines/components/nicEdit.jsx

```diff
@@ -56,14 +56,17 @@ class EditNICModal extends React.Component {
         let defaultNetworkSource;
         let currentSource;
         let availableSources = [];
+
```

Comment:
UPs I just noticed this is unrelated newline change, I hope that's ok. If there are review comment I will fix on the repush.